src/Elastic.Apm/BackendComm/CentralConfig/CentralConfigResponseParser.cs

```diff
@@ -136,30 +136,37 @@ internal class CentralConfigPayload
 		{
 			internal const string CaptureBodyContentTypesKey = "capture_body_content_types";
 			internal const string CaptureBodyKey = "capture_body";
-			internal const string TransactionMaxSpansKey = "transaction_max_spans";
-			internal const string TransactionSampleRateKey = "transaction_sample_rate";
 
 			internal const string CaptureHeadersKey = "capture_headers";
 			internal const string LogLevelKey = "log_level";
-			internal const string SpanFramesMinDurationKey = "span_frames_min_duration";
-			internal const string StackTraceLimitKey = "stack_trace_limit";
 			internal const string Recording = "recording";
 			internal const string SanitizeFieldNames = "sanitize_field_names";
+			internal const string SpanFramesMinDurationKey = "span_frames_min_duration";
+			internal const string StackTraceLimitKey = "stack_trace_limit";
+			internal const string TransactionIgnoreUrls = "transaction_ignore_urls";
+			internal const string TransactionMaxSpansKey = "transaction_max_spans";
+			internal const string TransactionSampleRateKey = "transaction_sample_rate";
 
 			internal static readonly ISet<string> SupportedOptions = new HashSet<string>
 			{
-				CaptureBodyKey, CaptureBodyContentTypesKey, TransactionMaxSpansKey, TransactionSampleRateKey,
-				CaptureHeadersKey, LogLevelKey, SpanFramesMinDurationKey, StackTraceLimitKey
+				CaptureBodyKey,
+				CaptureBodyContentTypesKey,
+				TransactionMaxSpansKey,
+				TransactionSampleRateKey,
+				CaptureHeadersKey,
+				LogLevelKey,
+				SpanFramesMinDurationKey,
+				StackTraceLimitKey
 			};
 
 			private readonly IDictionary<string, string> _keyValues;
 
 			public CentralConfigPayload(IDictionary<string, string> keyValues) => _keyValues = keyValues;
 
+			public string this[string key] => _keyValues.ContainsKey(key) ? _keyValues[key] : null;
```

Comment:
`TryGetValue()` and return the `out` parameter?

Reply:
Ok, changed it. I liked the 1-liner, but reading from the `IDictionary` only once is definitely better.